src/test/java/com/patientpal/backend/common/custommockuser/WithCustomMockUserPatient.java

```diff
@@ -0,0 +1,12 @@
+package com.patientpal.backend.common.custommockuser;
+
+import org.springframework.security.test.context.support.WithSecurityContext;
+import java.lang.annotation.Retention;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.lang.annotation.Retention' import. Should be before 'org.springframework.security.test.context.support.WithSecurityContext'.